src/entities/session-feedback.entity.ts

```diff
@@ -10,7 +10,7 @@ export class SessionFeedbackEntity extends BaseBloomEntity {
 
   @Column()
   sessionId: string;
-  @ManyToOne(() => SessionEntity, (sessionEntity) => sessionEntity.sessionUser, {
+  @ManyToOne(() => SessionEntity, (sessionEntity) => sessionEntity.sessionFeedback, {
```

Comment:
what caused this change? was this a previous bug?

Reply:
I think it was a typo. It was wrong. Weird it doesn't throw an error. Basically this line should have highlighted the many to one relationship between session feedback and sessions. See documentation https://orkhan.gitbook.io/typeorm/docs/many-to-one-one-to-many-relations.